doc/source/dev/build_a_job_runner.rst

```diff
@@ -0,0 +1,329 @@
+BUILD RUNNER FOR GALAXY
+-----------------------
+
+A walk through the steps of building a runner for Galaxy.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this tutorial, we would build the runner in a block by block fashion
+(like the building blocks), so we would divide the runner into
+components based on their function.
+
+Assuming you are familiar with setting up local install of galaxy and
+execution of galaxy. 
```

Comment:
We assume you are familiar with setting up and managing a local installation of Galaxy.